task_6_3.py

```diff
@@ -0,0 +1,7 @@
+list_str = []
+with open('C:/Users/Andrii/Desktop/learning_python.txt', 'r', encoding='utf-8') as lstr:
+    for line in lstr:
+        line = line.strip()
+        list_str.append(lstr.readlines())
+
+print(list_str)
```

Comment:
please check this code.
you started a loop. Then you strip your line. Afterthat you append **ALL** your lines to the list. you append all lines every iteration